magda-web-client/src/Components/Publisher/PublisherDetails.js

```diff
@@ -49,14 +49,16 @@ class PublisherDetails extends Component {
                             <h3 className="section-heading">Overview</h3>
                             <OverviewBox content={description} />
                         </div>
+                        <br />
                         <div>
-                            <Link
-                                to={`/search?organisation=${encodeURIComponent(
+                            <AUctaLink
```

Comment:
Using the react version means that it won't use the `react-router` link and will cause a full page reload right?

Maybe use the non-react version of the cta link (`<Link className="au-cta-link">`) instead?